plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java

```diff
@@ -118,6 +118,9 @@ private void addAppEngineLibrariesToBuildPath(IProject newProject,
       subMonitor.worked(1);
     }
     javaProject.setRawClasspath(newRawClasspath, monitor);
+    
+    Job job = new AppEngineLibraryContainerResolverJob("Initialize libraries", javaProject);
```

Comment:
AFAIK, unless this is a system job, `"Initialize libraries"` is a visible UI message.